shared/src/business/entities/cases/Case.js

```diff
@@ -2285,6 +2285,20 @@ const canAllowDocumentServiceForCase = rawCase => {
     return rawCase.canAllowDocumentService;
   }
 
+  return CASE_STATUS_TYPES.new !== rawCase.status;
+};
+
+/**
+ * determines whether or not to file automatically generated notices for notice of change of address
+ *
+ * @param {Object} rawCase the court case
+ * @returns {Boolean} whether or not we should automatically generate notices for a change of address
+ */
+const shouldGenerateNoticesForCase = rawCase => {
+  if (typeof rawCase.shouldGenerateNotices !== 'undefined') {
```

Comment:
I think I would prefer `if (rawCase.shouldGenerateNotices !== undefined) {` as checking `typeof` is probably an unnecessary step.